nvme-print.h

```diff
@@ -163,8 +164,8 @@ void nvme_show_supported_log(struct nvme_supported_log_pages *support,
 	const char *devname, enum nvme_print_flags flags);
 void nvme_show_error_log(struct nvme_error_log_page *err_log, int entries,
 	const char *devname, enum nvme_print_flags flags);
-void nvme_show_smart_log(struct nvme_smart_log *smart, unsigned int nsid,
-	const char *devname, enum nvme_print_flags flags);
+void nvme_show_smart_log(struct nvme_smart_log *smart, unsigned int nsid, const char *devname,
+			 enum nvme_print_flags flags, bool fahrenheit);
```

Comment:
I don't like to pass this as additional argument down. What about using the flags for this? Obviously, this means we would need to use a simpler type for flags here and not `enum nvme_print_flags`. This would also address the strange handling of the flags where we treat half of the time as enum and the rest of the time as bit field. A bit of code churn but again I think if we name this argument `flags` it should also be right type. What do you think?

Reply:
Yes right and agreed. Fixed the changes but the flag type will be fixed separately later. Thank you.